### PR TITLE
Remove local

### PR DESCRIPTION
**Pull request by kalantar**
Remove local commands
Is the iter8 part of https://github.com/iter8-tools/iter8/issues/1370. docs may also need updating.

### CODE REVIEW

File: abn/core/grpc_service_test.go [hunk 1]
```diff
@@ -244,3 +249,51 @@ func getMetric(a *abnapp.Application, track, metric string) *summarymetrics.Summ
...
+	return err
+}
+
+// nameFromKey returns the name from a key of the form "namespace/name"
+func nameFromKey(applicationKey string) string {
+	_, n := splitApplicationKey(applicationKey)
+	return n
+}
+
+// secretNameFromKey returns the name of the secret used to persist an applicatiob
```
Comment (Alan-Cha) [inline abn/core/grpc_service_test.go]:
```suggestion
// secretNameFromKey returns the name of the secret used to persist an application
```

File: abn/watcher/handler.go [hunk 2]
```diff
@@ -1,105 +1,320 @@
...
-		return false
+	application := getApplicationNameFromObjectName(obj.GetName())
+	namespace := obj.GetNamespace()
+	version, _ := getVersion(obj)
+	log.Logger.Tracef("handle called for %s/%s (%s))", namespace, application, version)
+
+	// get application configuration
+	appConfig := getApplicationConfig(namespace, application, config)
+	if appConfig == nil {
+		// we found a resource that is not part of an a/b/n test; ignore the object
```
Comment (Alan-Cha) [inline abn/watcher/handler.go]:
```suggestion
		// we found a resource that is not part of an A/B/n test; ignore the object
```

File: abn/watcher/handler.go [hunk 3]
```diff
@@ -1,105 +1,320 @@
...
+		log.Logger.Debugf("object for application %s/%s has no a/b/n configuration", namespace, application)
```
Comment (Alan-Cha) [inline abn/watcher/handler.go]:
```suggestion
		log.Logger.Debugf("object for application %s/%s has no A/B/n configuration", namespace, application)
```

File: abn/watcher/handler.go [hunk 4]
```diff
@@ -1,105 +1,320 @@
...
+		return true
+	}
+
+	// condition not True
+	return false
+}
+
+// TODO rewrite using NestedStringMap
+func getConditionStatus(obj *unstructured.Unstructured, conditionType string) (*string, error) {
+
```
Comment (Alan-Cha) [inline abn/watcher/handler.go]:
```suggestion
```

File: abn/watcher/handler.go [hunk 5]
```diff
@@ -1,105 +1,320 @@
...
+	if obj == nil {
+		return nil, errors.New("no object")
+	}
+
+	obj.GetNamespace()
```
Comment (Alan-Cha) [inline abn/watcher/handler.go]: Is there a purpose to this?

**Review by sriumcp — DISMISSED** [review]
lgtm

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**